src/platform/forms-system/src/js/web-component-patterns/relationshipToVeteranPattern.jsx

```diff
@@ -0,0 +1,62 @@
+import { radioUI, radioSchema } from './radioPattern';
+import VaTextInputField from '../web-component-fields/VaTextInputField';
+
+export const relationshipToVeteranUI = (
+  relationshipToVeteranTitle,
+  relationshipToVeteranOtherTitle,
+) => {
+  return {
+    relationshipToVeteran: radioUI({
+      title:
+        relationshipToVeteranTitle ??
+        'What’s your relationship to the veteran?',
+      labels: {
+        spouse: 'Spouse',
+        child: 'Child',
+        parent: 'Parent',
+        executor: 'Executor/Administrator of Estate',
+        other: 'A relationship not listed here',
+      },
+    }),
+    relationshipToVeteranOther: {
+      'ui:title':
+        relationshipToVeteranOtherTitle ??
+        'Since your relationship with the veteran was not listed, please describe it here',
+      'ui:webComponentField': VaTextInputField,
+      'ui:options': {
+        expandUnder: 'relationshipToVeteran',
+        expandUnderCondition: 'other',
+      },
+    },
+    'ui:options': {
+      expandedContentFocus: true,
```

Comment:
curious on the decision to put this here vs as a sibling to expandUnder. Is this more like the overall page setting? - also there may be a bug where if a page doesn't include this, but they try to use expandUnder the page doesn't seem to render.

Reply:
It was mainly because of how ObjectField and ExpandingGroup handle these fields. They get grouped together in `groupedProperties` array and then they get handed off to ExpandingGroup as a length 2 array of the primary field and the expanding under field. I couldn't really find a clean way of passing in a property from just the expanding under field. It was much easier to get it at the parent level because that's the scope for ExpandingGroup. 

All that being said, I think it's possible to do it the way you suggested. I could try to re-work it that way and see if it's a quicker than expected solution, but it wasn't when I tried it initially. I really just didn't think it was an issue to put it at the parent level because it _is_ functionality of the group of fields.

Reply:
Also there shouldn't be any bug or issue if this field isn't included...what issue are you thinking of? It would just be the regular "wizard" behavior without the field

Reply:
that seems reasonable. the only thing that might feel a little odd, is if you have other form fields also on the page, then it seems like that particular 'ui:options' pertains to everything on the page, not just that particular group right?

As for the bug I saw, I had tried adding a expandUnder to one of the fields on mockCheckboxAndTextInput, and got an error about expandedContentFocus, and the page wouldn't render so wasn't sure why.

Reply:
oh I see that this is for an encapsulated pattern, not a uiSchema level :)  then that seems fine!  my bad.  but yeah, I still see an issue when I try expandUnder for some reason on a different page element

Reply:
Okay let me experiment locally. Maybe I need to add a default value for `expandedContentFocus`